release-0.3.md

```diff
@@ -0,0 +1,47 @@
+# Contour 0.3
```

Comment:
Were you thinking to add this content to the readme? I assume that when you say you'll remove the file when you merge you don't mean you'll get rid of its contents ;). 
I'd recommend a new CHANGELOG.md file at the root; I'm happy to make the change when I copyedit if you like.

Reply:
I'll transpose this into the github release page, https://github.com/heptio/contour/releases

Reply:
got it, thanks